build/Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.3
-RUN apk --update --no-cache add \
-	ca-certificates
+
+RUN apk add --no-cache ca-certificates && \
+	rm -rf /var/cache/apk/*
```

Comment:
The example is for 3.1... So we need to remove the `--update` for the `--no-cache` to work: https://github.com/gliderlabs/docker-alpine/blob/master/docs/usage.md#disabling-cache

```
RUN apk --no-cache add \
	ca-certificates
```

I confirmed locally:

```
/ # ls -l /var/cache/apk/
total 0
```